src/components/Footer/FooterView.js

```diff
@@ -1,29 +1,42 @@
 import React, { useEffect, useState } from 'react';
 import { Footer } from 'bento-components';
-import FooterData, { getBackendVersion } from '../../bento/globalFooterData';
+import FooterData/* , { getBackendVersion } */from '../../bento/globalFooterData';
```

Comment:
@ogwuazors Can we remove comment later? Everything else looks good.

Reply:
yes, I'll make another branch for it